### PR TITLE
refactor(axelar-gateway)!: use more readable event symbols

### PR DESCRIPTION
**Pull request by milapsheth**
[AXE-6138]

[AXE-6138]: https://axelarnetwork.atlassian.net/browse/AXE-6138?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: contracts/axelar-gateway/src/event.rs [hunk 1]
```diff
@@ -20,23 +20,23 @@ pub(crate) fn call_contract(
...
-    env.events().publish(topics, message);
+    let topics = (Symbol::new(env, "message_approved"), message);
+    env.events().publish(topics, ());
 }
 
-pub(crate) fn execute_contract_call(env: &Env, message: Message) {
-    let topics = (symbol_short!("executed"),);
-    env.events().publish(topics, message);
+pub(crate) fn execute_message(env: &Env, message: Message) {
+    let topics = (Symbol::new(env, "message_executed"), message);
```
Comment (milapsheth) [inline contracts/axelar-gateway/src/event.rs:28]: moved `message` into the topic

File: contracts/axelar-gateway/src/test_gateway.rs [hunk 2]
```diff
@@ -10,6 +10,7 @@ use axelar_soroban_interfaces::types::Message;
 use axelar_soroban_std::{assert_contract_err, assert_invocation, assert_last_emitted_event};
 use soroban_sdk::testutils::BytesN as _;
 
+use soroban_sdk::Symbol;
 use soroban_sdk::{
     bytes, symbol_short,
```
Comment (ahramy) [inline contracts/axelar-gateway/src/test_gateway.rs:15]: unused import: `symbol_short`

File: contracts/axelar-gateway/src/testutils.rs [hunk 3]
```diff
@@ -7,7 +7,7 @@ use axelar_soroban_std::{assert_last_emitted_event, assert_ok};
 use ed25519_dalek::{Signature, Signer, SigningKey};
 use rand::Rng;
 
-use soroban_sdk::symbol_short;
+use soroban_sdk::{symbol_short, Symbol};
```
Comment (ahramy) [inline contracts/axelar-gateway/src/testutils.rs:10]: unused import: `symbol_short`

**Review by ahramy — APPROVED** [review]

File: contracts/axelar-gateway/src/test_gateway.rs (hunk 2 above)
Comment (milapsheth) [reply contracts/axelar-gateway/src/test_gateway.rs]: hmm, not sure why actions didn't complain. maybe take a look later to see if we need to enable certain lints

**Pull request merged**